docs/jwst/references_general/emi_reffile.inc

```diff
@@ -20,46 +20,82 @@ MIRI EMI reference files are in ASDF format.  The EMI
 reference file contains the frequencies for which the image will
 be corrected.
 Example file contents:
```

Comment:
Would a [literal block](https://docutils.sourceforge.io/docs/user/rst/quickref.html#literal-blocks) look ok?

I think if you undo this commit d07d15a6c5f8786ff701eca8c2cec766b70ab87d and make the `:` into a `::` add a new line after `::` and indent the tree you'll get a literal block that looks like:
![Uploading Screen Shot 2023-12-12 at 3.39.54 PM.png…]()

Reply:
Ha! @braingram beat me to it!